packages/webpack/test/fixtures/main/webpack.config.js

```diff
@@ -4,6 +4,11 @@ const HtmlWebpackPlugin = require('html-webpack-plugin')
 const path = require('path')
 
 const defaultLmOptions = {
+  scuttleGlobalThis: {
+    enabled: false,
+    exceptions: [],
+    scuttlerName: '',
```

Comment:
since the last two options are optional, can they just be omitted?

Reply:
`enabled: false` is default, so basically we can omit this diff all together, I just think being explicit here is worthwhile, do you disagree?

Reply:
asserting the defaults work as expected has more value imo

Reply:
it seems that the parent object is called `defaultLmOptions` and it indicates more (probably default) stuff, i prefer not to deviate from the intention of the original author (@naugtur thoughts?)

Reply:
these are defaults for tests. As opposed to test-specific setup. 
I agree there's value in asserting default settings work in the main test and then you provide that custom setting in specific tests as you do.

Reply:
is this better? 3d64180f9